controller/array_action/array_mediator_xiv.py

```diff
@@ -226,18 +254,24 @@ def copy_to_existing_volume_from_source(self, name, source_name, source_capacity
             logger.exception(ex)
             raise controller_errors.PermissionDeniedError("create vol : {0}".format(name))
 
-    def _get_vol_by_wwn(self, volume_id):
-        vol_by_wwn = self.client.cmd.vol_list(wwn=volume_id).as_single_element
-        if not vol_by_wwn:
-            raise controller_errors.ObjectNotFoundError(volume_id)
+    def _get_cli_object_by_wwn(self, volume_id, not_exist_err=False):
+        cli_object = self.client.cmd.vol_list(wwn=volume_id).as_single_element
+        if not cli_object:
+            if not_exist_err:
+                raise controller_errors.ObjectNotFoundError(volume_id)
+            return None
+        return cli_object
```

Comment:
can be [shortened](https://github.com/IBM/ibm-block-csi-driver/pull/253#discussion_r524270702)

Reply:
done